src/main/frontend/common/i18n/translations.ts

```diff
@@ -75,6 +75,7 @@ const DEFAULT_MESSAGES: ResourceBundle = {
   "Util.month": "{0} mo",
   "Util.year": "{0} yr",
   startedAgo: "Started {0} ago",
+  noBuilds: "No builds found",
```

Comment:
what do you think about matching with the build history widget text?

<img width="206" alt="image" src="https://github.com/user-attachments/assets/3ba70974-3f57-4748-a729-791fd5d83881" />


```suggestion
  noBuilds: "No builds",
```

Reply:
Fine by me, happy to be consistent with other parts of the UI